concepts.tex

```diff
@@ -726,18 +734,19 @@
 template <class T>
 concept bool StrictTotallyOrdered =
   EqualityComparable<T> &&
-  requires(const T a, const T b) {
-    { a < b } -> Boolean;
-    { a > b } -> Boolean;
-    { a <= b } -> Boolean;
-    { a >= b } -> Boolean;
+  requires(const remove_reference_t<T>& t,
+           const remove_reference_t<U>& u) {
+    { a < b }  -> Boolean&&;
+    { a > b }  -> Boolean&&;
+    { a <= b } -> Boolean&&;
+    { a >= b } -> Boolean&&;
```

Comment:
> Did I apply the cross-type changes to the single-type concept?

Ugh, no - that's what the PR actually says. I think we need to apply some "editorial" fixes here,